livre/core/data/MemoryUnit.h

```diff
@@ -81,15 +75,13 @@ class NoMemoryUnit : public MemoryUnit
 class ConstMemoryUnit : public MemoryUnit
 {
 public:
-    ConstMemoryUnit( const uint8_t* ptr, const size_t size );
+    explicit ConstMemoryUnit( const uint8_t* ptr );
```

Comment:
So as per our discussion, the whole MemoryUnit classes should be rewritten, but this is a bigger story. Meanwhile, this "cleanup" is OK as it highlights the underlying design issues.